ecl/sss/v1/approval_requests/results.go

```diff
@@ -6,11 +6,11 @@ import (
 )
 
 type Action struct {
-	Service string `json:"service"`
-	Region  string `json:"region"`
-	APIPath string `json:"api_path"`
-	Method  string `json:"method"`
-	Body    string `json:"body"`
+	Service string                 `json:"service"`
+	Region  string                 `json:"region"`
+	APIPath string                 `json:"api_path"`
+	Method  string                 `json:"method"`
+	Body    map[string]interface{} `json:"body"`
```

Comment:
Can you consider using `json.RawMessage`? It is more suitable for this field than map.
https://golang.org/pkg/encoding/json/#RawMessage

Reply:
I agree.I was worried that actions.body could contain strings as well as json objects.
Json.RawMessage worked fine even for strings.